features/bootstrap/FeatureContext.php

```diff
@@ -137,4 +164,65 @@ public function iSendARequestTo($method, $url, PyStringNode $body = null, $files
             $body !== null ? $body->getRaw() : null
         );
     }
+
+    /**
+     * Create resource if there are not created
+     * @Given Resource :resource :id exists
+     */
+    private function exists($resource, $id)
+    {
+        if (!strpos($id, $resource)) {
+            return false;
+        }
+
+        $this->request->send(
+            'GET',
+            $id,
+            [],
+            [],
+            null
+        );
+
+        return $this->getMinkContext()->getSession()->getStatusCode() !== 404;
+    }
+
+    /**
+     * @Given There are :resource :id
+     */
+    public function thereAreResource($resource, $id)
+    {
+        $id = explode(',', $id);
+
+        foreach ($id as $eachId) {
+            if (!$this->exists($resource, $eachId)) {
+                $helper = $resource . 'Helper';
+                $this->{$helper}->createResource();
+            }
+        }
+    }
+
+    private function relationExists(string $eachId1, string $resource2, string $id2)
+    {
+        // TODO check if relation exists
+    }
+
+    /**
+     * @Given There are :resource :id which have :resource2 :id2
+     */
+    public function thereAreResourceWhichHaveResource($resource, $id, $resource2, $id2)
+    {
+        $this->thereAreResource($resource, $id);
+        $this->thereAreResource($resource2, $id2);
+
+
+        $id = explode(',', $id);
+        $id2 = explode(',', $id2);
+
+        foreach ($id as $eachId1) {
+            if (!$this->relationsExists($eachId1, $resource2, $id2)) {
+                $helper = $resource . 'Helper';
+                $this->{$helper}->createRelationWith($eachId1, $resource2, $id2);
```

Comment:
Ok for using this but you should at least create an Interface for the RessourceHelper (`RessourceHelperInterface` to follow Symfony conventions) describing the role of the method.